tools/SchemaManager.Core/CommandOptions.cs

```diff
@@ -6,7 +6,7 @@
 using System;
 using System.CommandLine;
 
-namespace SchemaManager
+namespace SchemaManager.Core
```

Comment:
My thinking when I refactored stuff out to SchemaManager.Core was that the common logic for running the tool would be in .Core, but the entry points would be in separate projects (for example, Command Line would be in the SchemaManager project, while the code for applying it on startup is in the SqlServer.Api project).

Reply:
Yeah, makes sense! Also, I was debating myself too to reuse this file or not. Because in paas, its not going to be used as tool rather job, and we don't need all of the options like next. Also we might need a different parameter instead of connection-string. So I think I should create a separate file for paas command options. Thoughts?

Reply:
Reverted the change in last commit!

Reply:
Yup exactly, we'll have a separate file for those commands in that repo.